tap_db2/__init__.py

```diff
@@ -4,7 +4,7 @@
 from singer.catalog import Catalog
 from . import resolve, sync, discovery, common
 
-REQUIRED_CONFIG_KEYS = ["host", "port", "user", "password"]
```

Comment:
Need to leave port here

Reply:
The code doesn't require the port in order to function. Is there some other reason?

Reply:
It doesn't? https://github.com/singer-io/tap-db2/commit/71562f6fc9562c1244519b4c48049410aad8ae98#diff-2cbbde2899ce27cd1583a685b2d02688L55

Reply:
Nope. `config.get("db2_port")` -- `.get` returns `None` if the key doesn't exist. I also confirmed this since it's easier to run without the `port` in the configuration on the test ec2 instance.

Reply:
gotcha